app/cli/internal/action/attestation_add.go

```diff
@@ -20,12 +20,13 @@ import (
 	"errors"
 	"fmt"
 
+	"google.golang.org/grpc"
```

Comment:
Are you using any specific sorting plugin?

Reply:
Just `goimports --local`